hollow/src/main/java/com/netflix/hollow/api/consumer/HollowConsumer.java

```diff
@@ -1152,6 +1153,7 @@ public static class Builder<B extends HollowConsumer.Builder<B>> {
         protected HollowConsumer.ObjectLongevityDetector objectLongevityDetector = ObjectLongevityDetector.DEFAULT_DETECTOR;
         protected File localBlobStoreDir = null;
         protected boolean useExistingStaleSnapshot;
+        protected Duration existingBlobMaxAge;
```

Comment:
nit: for readability consider setting to -1 explicitly and a comment that this makes max age infinite

Reply:
nit: could init to -1 here to for readability

Reply:
Updated to -1s and added a comment.